lib/autoupdate/agent/installer.go

```diff
@@ -782,13 +786,5 @@ func (li *LocalInstaller) isLinked(versionDir string) (bool, error) {
 			return true, nil
 		}
 	}
-	linkData, err := readFileN(filepath.Join(li.LinkServiceDir, serviceName), maxServiceFileSize)
```

Comment:
Note: removing this logic fixes linked-version detection via `isLinked`. Two versions can have the same file contents.